src/types/IstioObjects.ts

```diff
@@ -80,13 +80,18 @@ export interface Pod {
   labels?: { [key: string]: string };
   createdAt: string;
   createdBy: Reference[];
+  containers?: ContainerInfo[];
   istioContainers?: ContainerInfo[];
   istioInitContainers?: ContainerInfo[];
   status: string;
   appLabel: boolean;
   versionLabel: boolean;
 }
 
+export interface PodLogs {
+  logs?: string;
```

Comment:
Since this is a really large string and it is more interesting than just a string how about assigning it its own type alias like `PodLogString`?

Reply:
Good idea.